public/kibi_timeline.js

```diff
@@ -237,23 +236,19 @@ define(function (require) {
                   const startValue = value;
                   const startRawValue = startRawFieldValue[i];
 
-                  //let content =
-                  //    '<div title="index: ' + indexId +
-                  //    ', startField: ' + params.startField +
-                  //    (params.endField ? ', endField: ' + params.endField : '') + '">' + labelValue +
-                  //    (params.useHighlight ? '<p class="tiny-txt">' + timelineHelper.pluckHighlights(hit, highlightTags) +
-                  //    '</p>' : '') + '</div>';
-                  const $itemscope = $scope.$new();
-                  $itemscope.contentDict = {
+                  const itemDict = {
                     indexId: indexId,
                     startField: params.startField,
                     endField: params.endField,
                     labelValue: labelValue,
                     useHighlight: params.useHighlight,
-                    highlight: timelineHelper.pluckHighlights(hit, highlightTags)
+                    highlight: timelineHelper.pluckHighlights(hit, highlightTags),
+                    groupColor: groupColor,
+                    startValue: startValue,
+                    endFieldValue: endFieldValue
                   };
-                  let content = $compile(itemTemplate)($itemscope)[0].innerHTML;
-                  console.log('INNER HTML:');
+
+                  const content = timelineHelper.createItemTemplate(i, itemDict);
                   console.log(content);
 
                   let style = `background-color: ${groupColor}; color: #fff;`;
```

Comment:
@szydan Seems I can't deprecate this `style` because I need to change the `groupColor` for every group.